.github/workflows/check-markdown-task.yml

```diff
@@ -0,0 +1,112 @@
+# Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/check-markdown-task.md
+name: Check Markdown
+
+env:
+  # See: https://github.com/actions/setup-node/#readme
+  NODE_VERSION: 12.x

```

Comment:
Better to use a more recent version like this one:
https://github.com/MatteoPologruto/arduino-language-server/blob/3150919e456b189f3ac0d5aa8c70ed24161e0c49/.github/workflows/check-markdown-task.yml#L6

Reply:
Also this version should already be preinstalled https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#language-and-runtime

Reply:
>Better to use a more recent version

@umbynos we should use a consistent version of Node.js for all aspects of the project. Unfortunately the action is currently configured to run with Node.js 12.x.

When a contributor is working on the project locally they are going to have that version of Node.js enabled, and that is also the version of Node.js that will run the Markdown tools when they execute the task locally. The CI should produce the same environment as the contributor will have on their local machine so that we can be sure the results will be the same.

Of course the ridiculously outdated version of Node.js used by the project must be updated, but that is a project for a different PR.

Reply:
you are right Per, I didn't notice that we use the same version in another action, thanks for pointing this out